models/kim_cnn/args.py

```diff
@@ -0,0 +1,23 @@
+import os
+import models.args
+
+
+def get_args():
+    parser = models.args.get_args()
+
+    parser.add_argument('--dataset', type=str, default='SST-1', choices=['SST-1', 'SST-2', 'Reuters', 'AAPD', 'IMDB', 'Yelp2014'])
```

Comment:
Is this case sensitive?

Reply:
Currently, I think so. Do you want everything to be in lowercase?